rest/views.py

```diff
@@ -89,8 +90,15 @@ class VocabularyList(ListCreateAPIView):
 
     def get_queryset(self):
         """Filter to vocabularies for a repository"""
```

Comment:
It means to return a list of vocabularies filtered by ownership by the repository